html/404.html

```diff
@@ -5,9 +5,9 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!-- Normalize.css makes HTML look consistent in all browsers -->
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/styles.css">
-    <link rel="stylesheet" href="css/404.css">
+    <link rel="stylesheet" href="/static/stylesheets/normalize.css">
+    <link rel="stylesheet" href="/static/stylesheets/styles.css">
+    <link rel="stylesheet" href="/static/stylesheets/404.css">
```

Comment:
@DanielekPark Please remove the `./html` directory from the repo since its not used with the service and doesn't need to be bundled with the app. As described above, you can add these files to `.gitignore` so you can use them locally.